generics/helpers/common_handler_v2.js

```diff
@@ -12,6 +12,8 @@ const rp = require('request-promise')
 const filesHelper = require(MODULES_BASE_PATH + '/cloud-services/files/helper')
 const axios = require('axios')
 const GotenbergConnection = require(SERVICES_BASE_PATH + '/gotenberg')
+const utilsConnection = require(GENERICS_FILES_PATH + '/helpers/utils')
```

Comment:
@MallanagoudaB we have global availability of utils. check how other utils functions are accessed. like that you should call the function